userguide/platform/userguide_deployment.adoc

```diff
@@ -124,11 +128,15 @@ In addition to the bus events, which are dispatched immediately, Kill Bill also
 
 The future notifications also rely on two tables: the `notifications` and `notifications_history`, and the mechanism to move processed entries is similar to what we described for the bus event.
 
-== Logging and GDPR
+== Logging and data governance
```

Comment:
I think words with length > 3 in titles need to be Capitalized -- at least this is what i was told in my early days when i was writing doc.

Reply:
Fixed in ebc0417d1ad3372951cfe7b642ac4b42e310c5fb :-)